examples/sai_dma_passthru.rs

```diff
@@ -60,11 +77,26 @@ fn main() -> ! {
     let ccdr = dp
         .RCC
         .constrain()
-        .use_hse(16.mhz())
-        .sys_ck(400.mhz())
+        .use_hse(HSE_CLOCK_MHZ)
+        .sys_ck(CLOCK_RATE_HZ)
+        .pclk1(PCLK_HZ) // DMA clock
+        // PLL1
+        .pll1_strategy(rcc::PllConfigStrategy::Iterative)
+        .pll1_p_ck(PLL1_P_HZ)
+        .pll1_q_ck(PLL1_Q_HZ)
+        .pll1_r_ck(PLL1_R_HZ)
+        // PLL2
+        .pll2_p_ck(PLL2_P_HZ) // Default adc_ker_ck_input
```

Comment:
Is there a reason for specifying all these PLL outputs here? It would be simpler to leave them out if they're not used for this example

Reply:
Sure, I can remove those. I put them in for debugging some stuff but they shouldn't be needed.